assets/js/blocks/cart-checkout/payment-methods/express-payment-methods.js

```diff
@@ -79,10 +80,10 @@ const ExpressPaymentMethods = () => {
 		( errorMessage ) => {
 			setPaymentStatus().error( errorMessage );
 			setExpressPaymentError( errorMessage );
-			setActivePaymentMethod( previousActivePaymentMethod.current );
-			if ( previousPaymentMethodData.current.isSavedToken ) {
```

Comment:
This code appears to have been added as a workaround for started/pristine data resetting. Saved methods should not be using `STARTED` so I've removed it.